src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/PathNodes.groovy

```diff
@@ -1,22 +1,46 @@
 package org.openkilda.functionaltests.helpers.model
 
+import org.openkilda.messaging.info.event.PathNode
 import org.openkilda.messaging.payload.flow.PathNodePayload
-import org.openkilda.northbound.dto.v2.flows.FlowPathV2
+import org.openkilda.northbound.dto.v2.flows.FlowPathV2.PathNodeV2
 
+import groovy.transform.Canonical
 import groovy.transform.ToString
 
+@Canonical
 @ToString(includeNames = true, includePackage = false)
 class PathNodes {
-    List<FlowPathV2.PathNodeV2> nodes
+    List<PathNodeV2> nodes
 
     PathNodes(List<PathNodePayload> nodes) {
         this.nodes = nodes.collect {
-            [it.inputPort ? new FlowPathV2.PathNodeV2(it.getSwitchId(), it.getInputPort(), null) : null,
-             it.outputPort ? new FlowPathV2.PathNodeV2(it.getSwitchId(), it.getOutputPort(), null) : null]
+            [it.inputPort ? new PathNodeV2(it.getSwitchId(), it.getInputPort(), null) : null,
+             it.outputPort ? new PathNodeV2(it.getSwitchId(), it.getOutputPort(), null) : null]
         }
                 .flatten()
                 .findAll()
     }
 
-    
+    List<PathNode> pathNodeView() {
```

Comment:
Methods which perform conversions to other types usually named `to`, e.g. `toPathNodes()`, `toPathNodesV2()`, etc.